k8s/prod-netmanager-frontend.yaml

```diff
@@ -0,0 +1,41 @@
+###########################################################
+# Start deployment of netmanager frontend
+###########################################################
+uiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airqo-netmanager-frontend
+  namespace: production
+spec:
+  selector:
+    matchLabels:
+      app: netmanager-ui
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: netmanager-ui
+    spec:
+      containers:
+        - name: netmanager-ui
+          image: us.gcr.io/airqo-250220/airqo-netmanager-frontend:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 80
+              name: netmanager-ui
+
+---
+kind: Service
+uiVersion: v1
+metadata:
+  name: airqo-netmanager-frontend-svc
+  namespace: production
+spec:
+  selector:
+    app: netmanager-ui
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+      nodePort: 30100
```

Comment:
Looks like the production nodePorts for both Analytics and Netmaager are the same here.

Reply:
oh... nice catch. I will work on it this evening. Thank you!

Reply:
this was resolved

Reply:
Awesome...thanks.